src/test/java/com/pi4j/driver/sensor/bmx280/Bmx280DriverTest.java

```diff
@@ -0,0 +1,21 @@
+package com.pi4j.driver.sensor.bmx280;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class Bmx280DriverTest {
+
+    @Test
+    public void testBmx280DriverUnrecognizedChipId() {
+        FakeI2CRegisterDataReaderWriter access = new FakeI2CRegisterDataReaderWriter(
+                FakeI2CRegisterDataReaderWriter.READ, Bmp280Constants.CHIP_ID, 123
+        );
+
+        assertThrows(
+                IllegalStateException.class,
+                () -> new Bmx280Driver(access),
```

Comment:
@FDelporte don't we have a mocking of the SPI, with which we can test the initialization?

Reply:
Would this be fine in a followup?